docs/documentation/stories/rc.0-update.md

```diff
@@ -0,0 +1,451 @@
+ `ng new upgrade-project --skip-install` and compare the differences.
+
+## @angular/cli
+
+Angular CLI can now be found on NPM under `@angular/cli` instead of `angular-cli`, and has a
+minimum requirement of Node 6.9.0 or higher, together with NPM 3 or higher.
+
+If you're using Angular CLI `beta.28` or less, you need to uninstall the `angular-cli` package.
+```bash
+npm uninstall -g angular-cli
+npm uninstall --save-dev angular-cli
+```
+
+To update Angular CLI to a new version, you must update both the global package and your project's
+local package.
+
+Global package:
+```bash
+npm uninstall -g @angular/cli
+npm cache clean
+npm install -g @angular/cli@latest
+```
+
+Local project package:
+```bash
+rm -rf node_modules dist # use rmdir on Windows
+npm install --save-dev @angular/cli@latest
+npm install
+```
+
+## One tsconfig per app
+
+CLI projects now use one tsconfig per app ([#4924](https://github.com/angular/angular-cli/pull/4924)).
+
+- `src/tsconfig.app.json`: configuration for the Angular app.
+```
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016",
+      "dom"
+    ],
+    "outDir": "../out-tsc/app",
+    "target": "es5",
+    "module": "es2015",
+    "baseUrl": "",
+    "types": []
+  },
+  "exclude": [
+    "test.ts",
+    "**/*.spec.ts"
+  ]
+}
+```
+- `src/tsconfig.spec.json`: configuration for the unit tests.
+```
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
+    ],
+    "outDir": "../out-tsc/spec",
+    "module": "commonjs",
+    "target": "es6",
+    "baseUrl": "",
+    "types": [
+      "jasmine",
+      "node"
+    ]
+  },
+  "files": [
+    "test.ts"
+  ],
+  "include": [
+    "**/*.spec.ts"
+  ]
+}
+```
+- `e2e/tsconfig.e2e.json`: configuration for the e2e tests.
+```
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
+    ],
+    "outDir": "../dist/out-tsc-e2e",
+    "module": "commonjs",
+    "target": "es6",
+    "types":[
+      "jasmine",
+      "node"
+    ]
+  }
+}
+```
+
+There is an additional root-level `tsconfig.json` that is used for IDE integration.
+```
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
+    ]
+  }
+}
+```
+
+You can delete `e2e/tsconfig.json` and `src/tsconfig.json` after adding these.
+
+Also update `.angular-cli.json` to use them inside `apps[0]`:
```

Comment:
I believe it's better to first introduce the name change from the angular-cli.json file to .angular-cli.json (see current line 177) and afterwards the updates inside .angular-cli.json (see current line 159).